packages/react/src/field/Field.tsx

```diff
@@ -0,0 +1,85 @@
+import * as RadixLabel from "@radix-ui/react-label";
+import { useId } from "@reach/auto-id";
+import {
+  type ComponentPropsWithRef,
+  type ReactElement,
+  cloneElement,
+  forwardRef,
+} from "react";
+
+import type { ExtendProps } from "../utils/ExtendProps";
+
+import { Box } from "../box";
+import { sprinkles } from "../sprinkles";
+import { Text } from "../text";
+type FormGroupProps = ExtendProps<
+  ComponentPropsWithRef<typeof Box>,
+  {
+    children: ReactElement;
+    description?: string;
+    disabled?: boolean;
+    error?: string;
+    id?: string;
+    label?: string;
+    required?: boolean;
+  }
+>;
+
+export const Field = forwardRef<HTMLDivElement, FormGroupProps>(
+  (
+    {
+      children,
+      description,
+      disabled,
+      error,
+      id: idProp,
+      label,
+      required,
+      ...props
+    },
+    ref,
+  ) => {
+    const id = useId(idProp);
+    return (
+      <Box display="flex" flexDirection="column" maxW="sm" ref={ref} {...props}>
+        {label && (
+          <RadixLabel.Root
+            className={sprinkles({
```

Comment:
let's use `<Text asChild>` here instead of manually using `className + sprinkles`

Reply:
i guess as a general rule `sprinkles()` should only really be used in `.css.ts` files - and in tsx we should use the native props